src/sqlancer/questdb/QuestDBOptions.java

```diff
@@ -0,0 +1,112 @@
+package sqlancer.questdb;
+
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.Parameters;
+import sqlancer.OracleFactory;
+import sqlancer.DBMSSpecificOptions;
+import sqlancer.common.oracle.CompositeTestOracle;
+import sqlancer.common.oracle.TestOracle;
+import sqlancer.questdb.QuestDBOptions.QuestDBOracleFactory;
+import sqlancer.questdb.QuestDBProvider.QuestDBGlobalState;
+
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+@Parameters(separators = "=", commandDescription = "QuestDB (default port: " + QuestDBOptions.DEFAULT_PORT
+        + " default host: " + QuestDBOptions.DEFAULT_HOST + ")")
+public class QuestDBOptions implements DBMSSpecificOptions<QuestDBOracleFactory> {
+    public static final String DEFAULT_HOST = "localhost";
+    public static final int DEFAULT_PORT = 8812;
+
+    @Parameter(names = "--oracle")
+    public QuestDBOracleFactory oracle = QuestDBOracleFactory.NOREC;
+
+    public enum QuestDBOracleFactory implements OracleFactory<QuestDBGlobalState> {
+        NOREC {
```

Comment:
Perhaps you can keep the enum empty for now? Later on, you can decide if you want to add one or multiple oracles.

Reply:
Yes, fixed this : )